CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased][unreleased]
 
+### Added
+
+- Import GIAS groups from csv data
```

Comment:
Might be worth say something like 'Added a new importer for group data from GIAS' as this reads like we have done an import?